src/guides/v2.3/ui_comp_guide/components/ui-date.md

```diff
@@ -55,6 +55,28 @@ Extends [`Abstract`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/M
 </form>
 ```
 
+## Validation
+
+There is the **validate-date** initialization rule, that describes the required **dateFormat** parameter, should be specified:
+
+```javascript
+'validate-date': [
```

Comment:
This JS snippet is not indented properly as I see it.

Reply:
Hi @BarnyShergold, thank you for your review! 

I have implemented your suggestions, could you please review and approve them?

Thank you! 😊